src/main/resources/templates/error.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    Something went wrong.
+</body>
+</html>
```

Comment:
I created a basic error page for the application so you can link to this from any controllers should you encounter any exceptions.

Source: https://martinfowler.com/bliki/TechnicalDebt.html